addon/utils/keyboard.ts

```diff
@@ -0,0 +1,38 @@
+const NUMERIC_ONLY: RegExp = /^\d$/i;
```

Comment:
I think this can be exported, it might be used from another component in the future